HIP/hip-850.md

```diff
@@ -0,0 +1,69 @@
+---
+hip: 850
+title: Enhancing Supply Key Functionality for NFT Updates in Treasury Wallet
+author: Patches (@HGraphPunks)
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Review
+created: 2024-01-02
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/660
+requested-by: HGraphPunks
+updated: 2024-01-24
+---
+
+## Abstract
+
+This HIP proposes enhancing the Supply Key functionality within the Hedera ecosystem, specifically enabling the Supply Key to update an NFT metadata field while the NFT is held in the treasury wallet. This ensures secure and controlled adaptability of NFTs, meeting dynamic needs while maintaining their integrity post-distribution.
+
+## Motivation
+
+Currently, there is a limitation in dynamically updating NFTs in a controlled manner. The proposed enhancement to the Supply Key functionality addresses this gap, allowing for secure and selective updates of NFTs within the treasury wallet, a feature particularly relevant for various evolving use cases in the Hedera ecosystem.
+
+## Rationale
+
+The enhancement focuses on granting the Supply Key the capability to update the data of an NFT serial number exclusively while it is in the treasury wallet. This approach ensures security and control, preventing unauthorized modifications once the NFT is distributed.
+
+## User Stories
+
+- As a game developer, I want to update game piece NFTs with player stats during gameplay, ensuring the data becomes immutable once the player regains ownership.
```

Comment:
These are great illustration of applications but maybe se should state it as a more general user story in terms of treasury account capability.
The user story will effectively be used as an E2E test cases to confirm the HIP.

Reply:
Same for example below

Reply:
@ty-swirldslabs this user story could be updated. Game dev is an application of the usage but a User story should be more general to the persona and activity